content/applications/marketing/email_marketing/lost_leads_email.rst

```diff
@@ -0,0 +1,321 @@
+
+.. _email_marketing/select_lost_reasons:
+
+Select appropriate lost reasons
+-------------------------------
+
+When a lead is marked as lost, Odoo recommends selecting a *Lost Reason* to indicate why the
+opportunity did not result in a sale. Doing so keeps the pipeline organized and reporting data
+accurate, and also generates potential to follow up with the lead in the future.
+
+If an existing *Lost Reason* is not applicable, users with the necessary permissions can create new
+ones, which means the lost reasons in a database vary from organization to organization and pipeline
+to pipeline. For more information on *Lost Reasons*, including the creation of them, refer to
+:doc:`../../sales/crm/pipeline/lost_opportunities`.
+
+By default, Odoo does include a few common reasons such as:
+
+- *Too expensive*
+- *We don't have people/skills*
+- *Not enough stock*
+
+When determining which reasons to include in a lost leads reactivation email, consider what the
+email is advertising in order pinpoint one or more relevant lost reasons. Then add a rule stating
+:guilabel:`Lost Reason` *does not contain* `_____` for every reason in the database **except** for
+the relevant one(s).
+
+.. example::
+   If the email advertises a selection of previously limited merchandise that is now back in stock,
+   it makes sense to target leads with the lost reason: *not enough stock*. To do so, add rules that
+   exclude all reasons except *not enough stock*.
+
+   .. image:: lost_leads_email/out-of-stock.png
+      :align: center
+      :alt: A list of filter rules that exclude all lost reasons except for Out of Stock.
+
+   If the email advertises a price reduction, it makes sense to target leads with the lost reason:
+   *too expensive*. To do so, add rules that exclude all reasons except *too expensive*.
+
+   .. image:: lost_leads_email/too-expensive.png
+      :align: center
+      :alt: A list of filter rules that exclude all lost reasons except for Too Expensive.
+
+.. _analyze_results:
+
+Analyze the results
+===================
+
+After sending a lost leads reactivation email, marketing teams can use the smart buttons along the
+top of the email to analyze the results and determine follow-up actions.
+
+Clicking on any of the smart buttons opens a list of the records matching that button's specific
+criteria.
+
+.. image:: lost_leads_email/smart-buttons.png
+   :align: center
+   :alt: The Mailing page of a sent email showing the smart buttons along the top of the page.
+
+The smart buttons include:
+
+- :guilabel:`Sent`: the total number of emails that were sent.
+- :guilabel:`Opened`: the percentage of recipients that opened the email.
+- :guilabel:`Replied`: the percentage of recipients that replied to the email.
+- :guilabel:`Clicked`: the click-through rate (%) of recipients that clicked on a link in the email.
+- :guilabel:`Leads`: the number of leads that have been created in the *CRM* pipeline as a result of
```

Comment:
Showing as opportuniites in 17.0 runbot
![2024-03-27_10-11](https://github.com/odoo/documentation/assets/91896312/f94c5235-6981-4b8b-9da3-fbd2df0d8b90)

Reply:
Wild, it shows up as Leads on mine and on internal 😆 
![Screenshot 2024-03-27 at 2 57 33 PM](https://github.com/odoo/documentation/assets/5496809/d0d8a67d-589c-4037-9110-f2820d14d158)

This might change based on how the CRM is set up (whether or not the **Leads** feature is activated), so I'll put `Leads/Opportunities` instead to cover both use cases